docs/contrib/Logger.rst

```diff
@@ -1,29 +1,94 @@
 Using the logging infrastructure
 ================================
 
-In order to ensure consistent messaging from the openroad application we
-have adopted spdlog as our logging infrastructure. We have a thin
-wrapper on top for extensibility. Whenever a message needs to be issued
-you will use one of the logging functions in the ‘ord’ namespace.
+OpenROAD provides spdlog as part of logging infrastructure in order to ensure a clear, consistent messaging and complete messaging interface. 
```

Comment:
I think it's more accurate to say "OpenROAD uses spdlog as its logging infrastructure"